vlib/v/tests/repl/error_exitasdfasdf.repl

```diff
@@ -1,6 +1,6 @@
 exitasdfasdf
 ===output===
-error: undefined ident: `exitasdfasdf`
+.vrepl.v:7:9: error: undefined ident: `exitasdfasdf`
```

Comment:
hm, weird ... the purpose of `convert_output` is to strip that prefix, but it did not work for some reason 🤔

Reply:
```v
		} else if line.contains('.vrepl.v:') {
			// Ensure that .vrepl.v: is at the start, ignore the path
			// This is needed to have stable .repl tests.
			idx := line.index('.vrepl.v:') or { panic(err) }
			content += endline_if_missed(line[idx..])
```

Reply:
Whether to filter out before and after messages and display only error messages?

Reply:
I found the reason, I'll change it.